### ISSUE TITLE
Synthetic event error in TopAppBarNavigationIcon

### ISSUE THREAD

**Issue by deilv** [jamesmfriedman/rmwc#536]
- What RMWC Version are you using [major.minor.patch]: 5.7.0

- Name your build system [Webpack, Rollup...]: Webpack

- Describe the bug with as much detail as possible:
Clicking the icon of &lt;TopAppBarNavigationIcon /&gt; results in a console warning

```
Warning: This synthetic event is reused for performance reasons. If you're seeing this, you're accessing the property `type` on a released/nullified synthetic event. This is set to null. If you must keep the original synthetic event around, use event.persist(). See https://fb.me/react-event-pooling for more information.
```

- What happened, and what was supposed to happen:
The warning appears when toggling state and is probably related to the icon being changed to the onIcon and vice versa. However, as you can see in the CodeSandBox example it happens even when onIcon is not used.

https://codesandbox.io/s/rmwc-sandbox-g3ou0

Since TopAppBarNavigationIcon is meant to be an instance of IconButton, I also added an IconButton to the right side of the bar which toggles without an error.

I will try to find some time to narrow it down to a specific line.

**Comment by jamesmfriedman** [comment]
The code sandbox you sent is set on the 6.0.0 alpha version actually. The issue doesn't appear to be in 5.7.x. Are you running v6?

**Comment by jamesmfriedman** [comment]
nvm, repoed it

**Comment by jamesmfriedman** [comment]
Released and fixed in 5.7.2

**Comment by deilv** [comment]
That was fast. Works like a charm, thanks :)
